src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/EmptyDictionary.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+
+namespace RadixDlt.NetworkGateway.PostgresIntegration.LedgerExtension;
+
+internal static class EmptyDictionary<TKey, TValue>
```

Comment:
Do we need that extension? what about using `ImmutableDictionary<TKey,TValue>.Empty` ?

Reply:
I knew there was something built-in but I somehow couldn't find it! Yes, obviously we should use `ImmutableDictionary<TKey,TValue>.Empty`.  It actually even naturally pushes the code into what I'd imagine it to look like long-term.